src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp

```diff
@@ -320,6 +320,9 @@ Containers::Optional<MeshData3D> TinyGltfImporter::doMesh3D(const UnsignedInt id
         const tinygltf::BufferView& bufferView = _d->model.bufferViews[accessor.bufferView];
         const tinygltf::Buffer& buffer = _d->model.buffers[bufferView.buffer];
 
+        const int stride = bufferView.byteStride;
+        CORRADE_ASSERT(stride == 0, "Trade::TinyGltfImporter::mesh3D(): non-zero strides for buffer views are unsupported.", {});
```

Comment:
... can this be a non-fatal failure? Like, returning `NullOpt` instead of blowing up :)

Reply:
Yes, already did that.